ports/renesas-ra/mpnetworkport.c

```diff
@@ -67,8 +67,8 @@ void mod_network_lwip_init(void) {
         timer_started = false;
     }
     // Start poll timer.
-    soft_timer_static_init(&network_timer, SOFT_TIMER_MODE_PERIODIC, 128, network_timer_callback);
-    soft_timer_reinsert(&network_timer, 128);
+    soft_timer_static_init(&network_timer, SOFT_TIMER_MODE_PERIODIC, 50, network_timer_callback);
+    soft_timer_reinsert(&network_timer, 50);
```

Comment:
I don't think dropping this poll interval to 50ms will improve TCP/UDP throughput, this polling is used for things like DHCP timeout.

Can you try iperf3 with the existing value of 128 here?

Reply:
This also polls the WiFi driver (like cyw43_poll), I did notice a significant increase when lower it, but not much below 50ms.

Reply:
This change is much more important than LWIP config, when reverting back to 128ms I lose 40% of the TCP throughput.

Reply:
OK.  I guess the esp-hosted driver needs more frequent polling (ideally it would interrupt the main processor when it needed polling).

Reply:
> ideally it would interrupt the main processor when it needed polling

It does, but it still needs to poll at that rate, I'm also looking at the firmware to see if there's room for improvements there.